price_pusher/src/index.ts

```diff
@@ -1,191 +1,9 @@
-  .parserConfiguration({
-    "parse-numbers": false,
-  })
-  .parseSync();
-
-const priceConfigs = readPriceConfigFile(argv.priceConfigFile);
-const priceItems = priceConfigs.map(({ id, alias }) => ({ id, alias }));
-
-// TODO: name ChainPricePusher -> IPricePusher in a clean up PR
-// TODO: update listeners to not depend on the whole priceConfig
-async function start({
-  sourcePriceListener,
-  targetPriceListener,
-  targetPricePusher,
-}: {
-  sourcePriceListener: IPriceListener;
-  targetPriceListener: IPriceListener;
-  targetPricePusher: ChainPricePusher;
-}) {
-  const handler = new Controller(
-    priceConfigs,
-    sourcePriceListener,
-    targetPriceListener,
-    targetPricePusher,
-    {
-      cooldownDuration: argv.cooldownDuration,
-    }
-  );
-
-  await handler.start();
-}
-
-const priceServiceConnection = new PriceServiceConnection(argv.priceEndpoint, {
-  logger: console,
-});
-
-const pythPriceListener = new PythPriceListener(
-  priceServiceConnection,
-  priceConfigs
-);
-
-function getNetworkPriceListener(network: string): IPriceListener {
-  switch (network) {
-    case "evm": {
-      const pythContractFactory = new PythContractFactory(
-        argv.endpoint,
-        fs.readFileSync(argv.mnemonicFile, "utf-8").trim(),
-        argv.pythContract
-      );
-
-      return new EvmPriceListener(pythContractFactory, priceItems, {
-        pollingFrequency: argv.pollingFrequency,
-      });
-    }
-
-    case "injective":
-      return new InjectivePriceListener(
-        argv.pythContract,
-        argv.endpoint,
-        priceItems,
-        { pollingFrequency: argv.pollingFrequency }
-      );
-    default:
-      throw new Error("invalid network");
-  }
-}
-
-function getNetworkPricePusher(network: string): ChainPricePusher {
-  const gasStation = getCustomGasStation(argv.customGasStation, argv.txSpeed);
-  switch (network) {
-    case "evm": {
-      const pythContractFactory = new PythContractFactory(
-        argv.endpoint,
-        fs.readFileSync(argv.mnemonicFile, "utf-8").trim(),
-        argv.pythContract
-      );
-      return new EvmPricePusher(
-        priceServiceConnection,
-        pythContractFactory.createPythContractWithPayer(),
-        gasStation
-      );
-    }
-    case "injective":
-      return new InjectivePricePusher(
-        priceServiceConnection,
-        argv.pythContract,
-        argv.endpoint,
-        fs.readFileSync(argv.mnemonicFile, "utf-8").trim()
-      );
-    default:
-      throw new Error("invalid network");
-  }
-}
-
-function getCustomGasStation(customGasStation?: number, txSpeed?: string) {
-  if (customGasStation && txSpeed) {
-    return new CustomGasStation(customGasStation, txSpeed);
-  }
-}
-
-start({
-  sourcePriceListener: pythPriceListener,
-  targetPriceListener: getNetworkPriceListener(argv.network),
-  targetPricePusher: getNetworkPricePusher(argv.network),
-});
+yargs(hideBin(process.argv)).command(evm).command(injective).help().argv;
```

Comment:
can you update the documentation / README to show the usage of the new command now

Reply:
yep, that's the next PR. cleanup and docs updates